pkg/machine/ignition.go

```diff
@@ -307,7 +307,12 @@ machine_enabled=true
 	delegateConf := `[Service]
 Delegate=memory pids cpu io
 `
-	subUID := `%s:100000:1000000`
+	// Prevent subUID from clashing with actual UID
+	subUID := 100000
+	if uid >= 100000 && uid <= 1100000 {
```

Comment:
1100000 is outside the range

```
%s:100000:1000000
```

```suggestion
	if uid >= 100000 && uid < 1100000 {
```